http-server-jetty/src/test/groovy/io/micronaut/servlet/jetty/JettyHttp2Spec.groovy

```diff
@@ -0,0 +1,77 @@
+package io.micronaut.servlet.jetty
+
+import groovy.transform.EqualsAndHashCode
+import io.micronaut.context.annotation.Property
+import io.micronaut.context.annotation.Requires
+import io.micronaut.core.annotation.Introspected
+import io.micronaut.http.*
+import io.micronaut.http.annotation.Body
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Header
+import io.micronaut.http.annotation.Post
+import io.micronaut.http.client.HttpClient
+import io.micronaut.http.client.HttpVersionSelection
+import io.micronaut.http.client.annotation.Client
+import io.micronaut.test.extensions.spock.annotation.MicronautTest
+import jakarta.inject.Inject
+import spock.lang.PendingFeature
+import spock.lang.Specification
+import spock.util.environment.OperatingSystem
+
+@MicronautTest
+@Property(name = 'spec.name', value = 'JettyHttp2PostSpec')
+@Property(name = "micronaut.server.http-version", value = "2.0")
+@Property(name = "micronaut.ssl.enabled", value = "true")
+@Property(name = "micronaut.ssl.build-self-signed", value = "true")
+class JettyHttp2Spec extends Specification {
+    @Inject
+    @Client(
+            value = "/",
+            alpnModes = HttpVersionSelection.ALPN_HTTP_2)
+    HttpClient client
+
+    @PendingFeature(reason = "Conscript configuration not supported")
+    void "test simple post request with JSON over http/2"() {
```

Comment:
this is disabled because I could find no way to test the setup of conscript + HTTP2 over TLS

Reply:
i thought this was only an ARM issue?

Reply:
I couldn't get it passing on CI, mystery to me. Seems like conscript conflicts with creating the SSL client config or something